stable/stolon/templates/metrics-service.yaml

```diff
@@ -11,8 +11,14 @@ metadata:
     component: "all"
 spec:
   ports:
+{{ if .Values.serviceMonitor.enabled }}
     - name: metrics
       port: {{ .Values.ports.metrics.containerPort }}
+{{ end }}
+{{ if .Values.keeper.postgresExporter.enabled }}
+    - name: pg-metrics
```

Comment:
Why do you add `pg-metrics` to both `metrics-service.yaml` and `keeper-headless-service.yaml` services?

Also, `metrics-service.yaml` selects all the pods, not only keepers, so this seems wrong.